praw/models/reddit/mixins/inboxable.py

```diff
@@ -7,7 +7,20 @@ class InboxableMixin(object):
     """Interface for RedditBase classes that originate from the inbox."""
 
     def block(self):
-        """Block the user who sent the item."""
+        """Block the user who sent the item.
+
+        Example usage:
+
+        .. code:: python
+
+           comment = reddit.comment('dkk4qjd')
+           comment.block()
```

Comment:
I believe this will only work if the comment is a reply to you or tags you (or otherwise ends up in your inbox). If that is true, it would be nice to have a note here in documentation.

Reply:
Added